Please add instructions on how to install dependencies

For go newbies (like myself) it is quite daunting how go handle package dependencies. What I've tried seems not to be sufficient:
```
export GOPATH=$(pwd)
go get github.com/tools/godep
./bin/godep get ./...
```

I got some issues like having to download dependencies with go get because godep did not find them:
```
src/github.com/onsi/gomega/ghttp/handlers.go:12:2: cannot find package "github.com/golang/protobuf/proto" in any of:
	/usr/lib/go/src/github.com/golang/protobuf/proto (from $GOROOT)
	/home/rm/src/arduino-create-agent/src/github.com/golang/protobuf/proto (from $GOPATH)
godep: exit status 1
rm@rieux:~/src/arduino-create-agent$ GOPATH=$(pwd) ./bin/godep get ./...
godep: Package (golang.org/x/sys/unix) not found
rm@rieux:~/src/arduino-create-agent$ GOPATH=$(pwd) go get golang.org/x/sys/unix
rm@rieux:~/src/arduino-create-agent$ GOPATH=$(pwd) ./bin/godep get ./...
godep: Package (github.com/getlantern/filepersist) not found
rm@rieux:~/src/arduino-create-agent$ GOPATH=$(pwd) go get github.com/getlantern/filepersist
rm@rieux:~/src/arduino-create-agent$ GOPATH=$(pwd) ./bin/godep get ./...
godep: Package (github.com/mattn/go-isatty) not found
rm@rieux:~/src/arduino-create-agent$ GOPATH=$(pwd) go get github.com/mattn/go-isatty
rm@rieux:~/src/arduino-create-agent$ GOPATH=$(pwd) ./bin/godep get ./...
godep: exec: "godep": executable file not found in $PATH
```

--- a/README.md
+++ b/README.md
@@ -21,6 +21,14 @@ Please use the current latest version:
 
 ## Compiling
 
+From the project root dir executing:
+```
+export GOPATH=$PWD
+go get
+go build
+```
+will build the `arduino-create-agent` binary.
+
 `compile_webidebridge.sh` contains the cross-platform script we use to deploy the agent for all the supported platforms; it needs to be adjusted as per your `go` installation paths and OS.
 
 You can use `bootstrapPlatforms` function to compile the needed CGO-enabled environment

